autoreject/autoreject.py

```diff
@@ -664,7 +664,7 @@ def _interpolate_bad_epochs(
             desc='Repairing epochs',
             position=pos, leave=True, verbose=verbose):
         epoch = epochs[epoch_idx]
-        epoch.info['bads'] = interp_chs
+        epoch.info['bads'] += interp_chs
```

Comment:
okay thinking more about this. I think the right fix is to use `picks` to ignore the user marked bad channel. The user is free to interpolate later if they want. This will affect also the estimation phase in `autoreject`.